pkg/parser/ini/ini_test.go

```diff
@@ -0,0 +1,29 @@
+package ini
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestIniParser(t *testing.T) {
+	parser := &Parser{
+		FileName: "sample.ini",
+	}
+
+	var input interface{}
+	err := parser.Unmarshal(nil, &input)
+	if err != nil {
+		t.Fatalf("parser should not have thrown an error: %v", err)
+	}
+
+	if input == nil {
+		t.Error("There should be information parsed but its nil")
+	}
+
+	inputMap := input.(map[string]interface{})
+	item := inputMap["Local Varaibles"]
+	fmt.Println(item.(map[string]interface{}))
```

Comment:
I guess this is a leftover from debugging locally. You can remove fmt entirely from the test.

Reply:
Nice catch @Blokje5, removing it.